docs/04_publishing.md

```diff
@@ -87,8 +87,17 @@ The general recommendation is to have more than 1,000 and fewer than 200,000 fil
 configurations of software are located in their own separate subdirectory (as is common with a software installation
 tool like [EasyBuild](https://easybuild.io)).
 
+
 Making nested catalogs manually can be done in two ways, which we will describe in more detail.
 
+!!! note "Exceeding the limit"
+
+    In case your catalog does grow larger than the recommended limit of 200,000 entries, you will get a warning when publishing new changes:
```

Comment:
I think the warning will pop up for *any* catalog file?

```suggestion
  In case a catalog file does grow larger than the recommended limit of 200,000 entries, you will get a warning when publishing new changes:
```

Reply:
True, thanks!